### PR TITLE
Add Steps component to `from-gatsby.mdx`

### PR DESCRIPTION
**Pull request by Egpereira**
# Add Steps component to `from-gatsby.mdx`

Adds the `Step` component where needed.

### CODE REVIEW

**Comment by DuanBoomer** [comment]
!coauthor

File: src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -393,6 +394,7 @@ This example converts the main project layout (`layout.js`) from Gatsby's blog s
 
 This page layout shows one header when visiting the home page, and a different header with a link back to Home for all other pages.
 
+<Steps>
 1. Identify the return() JSX.
```
Comment (DuanBoomer) [inline src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx:398]: change this to the following
Identify the `return()` JSX.

**Comment by DuanBoomer** [comment]
other than that the `<Steps>` component is good to go.
LGTM 🔥

**Review by DuanBoomer — APPROVED** [review]
LGTM

File: src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx (hunk 1 above)
Comment (TheOtterlord) [inline src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx:398]:
```suggestion
1. Identify the `return()` JSX.
```

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**